account_analytic_distribution_manual/tests/test_analytic_distribution_manual.py

```diff
@@ -24,3 +25,39 @@ def test_copy_manual_distribution(self):
                     "name": "Manual Distribution 1",
                 }
             )
+
+    def test_manual_distribution_analytic_distribution_text(self):
+        aa_1 = self.analytic_account_a1
+        aa_2 = self.analytic_account_a2
+        self.assertEqual(
+            self.distribution_1.analytic_distribution_text,
+            str({aa_1.name: 40.0, aa_2.name: 60.0}),
```

Comment:
You must hardcode this result for proper test asserts. If not, you are creating business code inside the test code.

Reply:
I mean:

```suggestion
            '{"blabla": 40.0, "blabla2": 60.0}',
```